lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,20 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = CreditRequirement.get_course_requirement(
+                course_key, "reverification", checkpoints[0].checkpoint_name
+            )
+            if credit_requirement is not None:
+                try:
+                    CreditRequirementStatus.add_requirement_status(
+                        attempt.user.username, credit_requirement, "satisfied"
+                    )
+                except Exception as exp:  # pylint: disable=broad-except
+                    log.warn(exp)
```

Comment:
Make this exception logging more descriptive by providing the description string what is this exception all about